tools/ovc/openvino/tools/ovc/telemetry_utils.py

```diff
@@ -15,6 +15,12 @@
 except ImportError:
     import openvino.tools.ovc.telemetry_stub as tm
 
+def is_optimum():
+    import traceback
```

Comment:
can we also extract version of optimum triggering OpenVINO?

Reply:
Yes, done.